index.js

```diff
@@ -76,7 +76,12 @@ Imports.module = function (source, params, verbose) {
     if ( 'object' == typeof params && !params.length ) {
         for ( var key in params ) {
             if ( params.hasOwnProperty(key) ) {
-                eval('var ' + key + ' = params[key];');
+                // since `key` is a `variable name`, it's needed verifying the `variable name`
```

Comment:
### This comment is a little long and the grammar is not right.

```javascript
// since `key` is a `variable name`, it's needed verifying the `variable name`
```

_**change it to:**_ 👇

```javascript
// validating the syntax of `key` since it's a variable name
```

Reply:
Thanks for the correction :smile:

Mik